### ISSUE TITLE
Question: About second training stage with model loading

### ISSUE THREAD

**Issue by xforceco** [OpenNMT/OpenNMT-py#602]
Let's say I have a model file trained after `15` epochs, trained with `learning_rate 1` and `start_decay_at 8`.
Now, I want to loading this model and train another `5` epochs with `learning_rate 1` and no decay, what should I do?

Using `train_from` and set `start_decay_at 999` and `epochs 20` does not work in the second training stage. (learning rate is not `1` in `epoch 16`)

**Comment by srush** [comment]
Is the problem that it starts from the earlier learning rate?

**Comment by srush** [comment]
Oh I see. Short term fix is to add a `False` to the top of this if statement. We default to continue using the same optimizer. We'll add an option to use a new optimizer.

https://github.com/OpenNMT/OpenNMT-py/blob/master/train.py#L367

**Comment by xforceco** [comment]
Thank you for adding that.
Currently I am hard coding the `self.lr` in `update_learning_rate` in `Optim.py`

**Comment by srush** [comment]
That works too :)
